lib/components/user/places/favorite-place-row.js

```diff
@@ -54,29 +58,36 @@ const DeletePlaceholder = styled.span`
   ${deleteCss}
 `
 
-const MESSAGES = {
-  DELETE: 'Delete this place',
-  EDIT: 'Edit this place'
-}
-
 /**
  * Renders a clickable button for editing a user's favorite place,
  * and lets the user delete the place.
  */
 const FavoritePlaceRow = ({ isFixed, onDelete, path, place }) => {
+  const intl = useIntl()
   if (place) {
     const { address, icon, name, type } = place
+    const editMessage = intl.formatMessage({id: 'components.FavoritePlaceRow.editThisPlace'})
+    const deleteMessage = intl.formatMessage({id: 'components.FavoritePlaceRow.deleteThisPlace'})
     return (
       <Container>
         <LinkContainerWithQuery to={path}>
           <PlaceButton
-            aria-label={MESSAGES.EDIT}
-            title={MESSAGES.EDIT}
+            aria-label={editMessage}
+            title={editMessage}
           >
             <GreyIcon name={icon} size='2x' />
             <PlaceContent>
               {name && <span>{name}</span>}
-              <span>{address || `Set your ${type} address`}</span>
+              <span>
+                {address || (
+                  <FormattedMessage
+                    id='components.FavoritePlaceRow.setAddressForPlaceType'
+                    values={{
+                      placeType: intl.formatMessage({id: `common.places.${type}`})
```

Comment:
Depending on if template strings for id generation are considered good practice, I would either have this match all the others or adjust all the others.

Reply:
Good catch, this should match the others.